src/plugins/intel_gpu/include/intel_gpu/plugin/primitives_list.hpp

```diff
@@ -119,7 +119,7 @@ REGISTER_FACTORY(v1, LogicalXor);
 REGISTER_FACTORY(v1, MaxPool);
 REGISTER_FACTORY(v1, Maximum);
 REGISTER_FACTORY(v1, Minimum);
-REGISTER_FACTORY(v1, Multiply);
+// REGISTER_FACTORY(v1, Multiply);
```

Comment:
```suggestion
REGISTER_FACTORY(v1, Multiply);
```
I believe it was done for demonstration/test purposes and it is not longer needed.